src/containers/Teams/index.tsx

```diff
@@ -87,26 +91,18 @@ const Teams: FC = () => {
   }, [history, teamParam]);
 
   useEffect(() => {
-    const getFilteredMembers = (): TeamMember[] => {
-      const teamLeads: TeamMember[] = [];
-      const otherMembers: TeamMember[] = [];
-      teamMembers.forEach((member) => {
-        const {teams} = member;
-        const matchingTeam = teams.find(({title}) => title.toLowerCase() === teamFilter.toLowerCase());
-        if (matchingTeam) {
-          if (matchingTeam.isLead) {
-            teamLeads.push({...member, isLead: true});
-          } else {
-            otherMembers.push({...member, isLead: false});
-          }
-        }
-      });
-      return teamLeads.concat(otherMembers);
-    };
+    const getFilteredMembers = () =>
+      teamMembers.filter((teamMember) => teamMember.team.toLowerCase() === teamFilter.toLowerCase());
+
     const members = teamFilter === TeamName.all ? teamMembers : getFilteredMembers();
     const sortedMembers = sortTeamMembers(members);
-    setFilteredMembers(sortedMembers);
-  }, [teamFilter]);
+
+    if (queryTitle) {
+      setFilteredMembers(sortedMembers.filter((member) => member.title.toLowerCase() === queryTitle.toLowerCase()));
```

Comment:
what is this change for? how can we set the `queryTitle` within our website? These changes are causing quite a few merge conflicts to https://github.com/thenewboston-developers/Website/pull/1663 but not sure to follow this implementation

Reply:
what's the use case for this? it seems like only if we add `title` query param manually in the url, then we will filter the teams based on member

Reply:
ok saw ur explanation above, ignore this